models/clm/work/input.nml

```diff
@@ -306,21 +368,21 @@
    loc_of_interest      =  90.0,  41.00, 0.35
 
 &model_mod_check_nml
-   input_state_files     = 'clm_restart.nc', 'clm_history.nc', 'clm_vector_history.nc'
+   input_state_files     = 'clm5_tag34.clm2_0001.r.nc', 'clm5_tag34.clm2_0001.h0.nc', 'clm5_tag34.clm2_0001.h2.nc'
    output_state_files    = 'mmc_output1.nc', 'mmc_output2.nc', 'mmc_output3.nc'
    test1thru             = 0
-   run_tests             = 1, 2, 3, 4, 7
-   x_ind                 = 720
-   loc_of_interest       = 254.727854, 39.9768545, 0.35
-   quantity_of_interest  = 'QTY_LEAF_CARBON'
+   run_tests             = 1, 2, 3, 4
+   x_ind                 = 482189
+   loc_of_interest       = 255.0, 50.0, 0.001
+   quantity_of_interest  = 'QTY_SOIL_TEMPERATURE'
    interp_test_vertcoord = 'VERTISHEIGHT'
    interp_test_lonrange  = 320.0, 90.0
    interp_test_dlon      = 1.0
    interp_test_latrange  = 0.0, 75.0
    interp_test_dlat      = 1.0
    interp_test_vertrange = 1.0, 2.0
    interp_test_dvert     = 1.0
-   verbose               = .false.
+   verbose               = .true.
```

Comment:
i suspect you don't want to leave verbose as true.  i think it prints out the entire state location info if this is true?  or maybe it does that anyway for some tests.  if this is helpful, leave it in.